pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -54,79 +55,109 @@ type HyperConvergedConfig struct {
 	NodePlacement *sdkapi.NodePlacement `json:"nodePlacement,omitempty"`
 }
 
+type FeatureGate *bool
+
 // HyperConvergedFeatureGates is a set of optional feature gates to enable or disable new features that are not enabled
 // by default yet.
 // +optional
 // +k8s:openapi-gen=true
 // +kubebuilder:default={}
 type HyperConvergedFeatureGates struct {
+	// +optional
```

Comment:
should we also change `+kubebuilder:default={sriovLiveMigration: false, hotplugVolumes: false, gpu: false, hostDevices: false, withHostPassthroughCPU: false, withHostModelCPU: true, hypervStrictCheck: true}` on line 44?

Reply:
you mean `true` for all of them, I guess

Reply:
yes, right, true for all the new ones